change/monorepo-scripts-3c5e63c1-709c-4d5d-af40-308b1d3df4ce.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Literally none",
+  "packageName": "monorepo-scripts",
+  "email": "eloy.de.enige@gmail.com",
+  "dependentChangeType": "none"
+}
```

Comment:
@kenotron It keeps thinking I made changes to this, any idea why?

Reply:
Flip that project to "private: true" perhaps would make it stop :)